sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1,1320 +1,1365 @@
-* SHAREPOINT:CONNECTIVITY
-* SHAREPOINT:CONNECTIVITY
+==== Throws
 * SHAREPOINT:RETRY_EXHAUSTED
 * SHAREPOINT:UNKNOWN
+* SHAREPOINT:CONNECTIVITY
 
-[[fileCheckOut]]
-== File Check Out
 
+[[fileCheckOut]]
+=== File Check Out
 `<sharepoint:file-check-out>`
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+| Configuration | String | The name of the configuration to use. | | x
+| File Server Relative Url a| String |  |  | x
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
 |===
 
-=== For Configurations
-
-* sharepoint-online
-* sharepoint-onpremise
 
-=== Throws
+=== Associated Configurations
+* <<sharepoint-online>>
+* <<sharepoint-onpremise>>
 
-* SHAREPOINT:CONNECTIVITY
-* SHAREPOINT:CONNECTIVITY
+==== Throws
 * SHAREPOINT:RETRY_EXHAUSTED
 * SHAREPOINT:UNKNOWN
+* SHAREPOINT:CONNECTIVITY
 
-[[fileCopyTo]]
-== File Copy To
 
+[[fileCopyTo]]
+=== File Copy To
 `<sharepoint:file-copy-to>`
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
-| New File Server Relative Url a| String | | |x
-| Overwrite a| Boolean | | false |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+| Configuration | String | The name of the configuration to use. | | x
+| File Server Relative Url a| String |  |  | x
+| New File Server Relative Url a| String |  |  | x
+| Overwrite a| Boolean |  |  false |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
 |===
 
-=== For Configurations
 
-* sharepoint-online
-* sharepoint-onpremise
+=== Associated Configurations
+* <<sharepoint-online>>
+* <<sharepoint-onpremise>>
 
-=== Throws
-
-* SHAREPOINT:CONNECTIVITY
-* SHAREPOINT:CONNECTIVITY
+==== Throws
 * SHAREPOINT:RETRY_EXHAUSTED
 * SHAREPOINT:UNKNOWN
+* SHAREPOINT:CONNECTIVITY
 
-[[fileDelete]]
-== File Delete
 
+[[fileDelete]]
+=== File Delete
 `<sharepoint:file-delete>`
 
-=== Parameters
 
+Deletes a file.
+
+
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+| Configuration | String | The name of the configuration to use. | | x
```

Comment:
This is not important. I was only removing the period for consistency's sake. I'll quit commenting on it though.